frontend/public/actions/features.ts

```diff
@@ -252,6 +254,19 @@ const detectConsoleLinks = (dispatch) =>
     },
   );
 
+const detectIndependentMode = (dispatch) => {
+  k8sGet(OCSServiceModel, 'ocs-storagecluster', 'openshift-storage')
+    .then(() => {
+      dispatch(setFlag(FLAGS.INDEPENDENT, true));
+      // eslint-disable-next-line no-console
+      console.log('Independent mode is active');
+    })
+    .catch((err) => {
+      _.get(err, 'response.status') === 404
+        ? dispatch(setFlag(FLAGS.INDEPENDENT, false))
+        : handleError(err, FLAGS.INDEPENDENT, dispatch, detectIndependentMode);
+    });
+};
```

Comment:
@rawagner would it possible to push it here through any extension point?

Reply:
we dont have any, so you will need to create a new one